src/mono/nuget/Microsoft.NET.Sdk.WebAssembly.Pack/build/Microsoft.NET.Sdk.WebAssembly.Browser.targets

```diff
@@ -19,6 +19,8 @@ Copyright (c) .NET Foundation. All rights reserved.
     <RunCommand Condition="'$(RunCommand)' == ''">dotnet</RunCommand>
 
     <WasmAppHostDir>$([MSBuild]::NormalizeDirectory($(MSBuildThisFileDirectory), '..', 'WasmAppHost'))</WasmAppHostDir>
+    <WasmBuildBootJsonFileName Condition="'$(WasmBuildBootJsonFileName)'==''">blazor.boot.json</WasmBuildBootJsonFileName>
+    <WasmBuildPublishBootJsonFileName Condition="'$(WasmBuildPublishBootJsonFileName)'==''">blazor.publish.boot.json</WasmBuildPublishBootJsonFileName>
```

Comment:
Since this property is used only for intermediate file, we can compute the file name for publish based on `WasmBootConfigFileName` and skip the need property, or at least for "public property". Something like `publish.$(WasmBootConfigFileName)` should be enough